docs/example-scenario/infrastructure/multi-tier-app-disaster-recovery.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Highly available (HA) and disaster recovery (DR) configured multi-tier web application
+description: Build a highly available and disaster recovery configured multi-tier web application on Azure using Azure virtual machines, Availability sets, Availability zones, Azure Site Recovery and Azure Traffic Manager
+author: sujayt
+ms.date: 11/16/2018
+---
+
+# Build a highly available and disaster recovery configured multi-tier web application
+
+This example scenario is applicable to any industry that have a need to deploy resilient multi-tier applications with high availability and disaster recovery configured. In this scenario, the application consists of 3 layers.
+
+- Web tier - This is the top most layer with user interface. This translates inputs and outputs into something user understands.
+- Business tier - This layer processed the user interactions and makes logical decisions about the next steps. It acts as a connection between the web tier and the data tier
+- Data tier - This is the layer where the data is stored. Typically, a database or a file storage is used.
+
+Example application scenarios include any mission critical application running Windows or Linux operating systems. This can be a standard application such as SAP and SharePoint or a custom Line of Business (LOB) application.
+
+## Relevant use cases
+
+Other relevant use cases include:
+
+* Deploy highly resilient standard applications such as SAP and SharePoint
+* Dsign business continuty and disaster recovery plan for Line of Business (LOB) applications
+* Configure disaster recovery (DR) and perform DR drills for compliance needs
+
+## Architecture
+
+This scenario covers a multi-tier application that uses ASP.NET and Microsoft SQL Server. In Azure regions supporting Azure availabilit zones, you can deploy your VMs in source region across availability zones and replicate the VMs to diaster recovry (DR) target region. In Azure regions not supporting availability zones, you can deploy your VMs within an availability set and replicate the VMs to disaster recovery (DR) target region.
```

Comment:
Is there a link we can put here which shows the regions that support AZ's?

Reply:
added a link